packages/cli/src/lib/live-server/index.js

```diff
@@ -382,11 +365,22 @@ LiveServer.start = function(options) {
       LiveServer.activeTabs = LiveServer.activeTabs.filter(tab => tab.client !== ws);
     };
 
-    // CHANGED: Added line to record tab client socket on creation
+    // CHANGED: Enhanced client websocket addition process to record the client as an active tab entry
     const reqUrl = path.dirname(request.url);
-    const tab = LiveServer.activeTabs.find(tab => tab.url === reqUrl && !tab.client);
-    tab.client = ws;
-    tab.isReloading = false;
+    // Guard clause for MarkBind's _include_ files, no need to recognize as an active tab
+    if (reqUrl.endsWith('._include_.html')) {
```

Comment:
shouldn't be needed with the socket approach as well since its not a page

Reply:
Ah I see, alright.